src/pages/experiments/[experimentId]/data-exploration/components/cell-sets-tool/CellSetsTool.jsx

```diff
@@ -72,25 +101,71 @@ const CellSetsTool = (props) => {
       );
     }
 
-    return (
-      <>
-        <Space style={{ width: '100%' }}>
-          <Tooltip title='Reset clusters to the initial state'>
-            <Button type='primary' size='small' onClick={recluster}>Reset Clusters</Button>
+    let operations = null;
+    const numSelected = getNumberOfCellsSelected();
+
+    if (numSelected) {
+      operations = (
+        <Space>
+          <Text type='secondary'>
+            {numSelected}
+            {' '}
+            cell
+            {numSelected === 1 ? '' : 's'}
+            {' '}
+            selected
+          </Text>
+          {/* <Tooltip title='Create complement of selected cells'>
+            <Button type='dashed' icon={<BlockOutlined />} size='small' />
+          </Tooltip>
+
+          <Tooltip title='Create intersection of selected cells'>
+            <Button type='dashed' icon={<SplitCellsOutlined />} size='small' />
           </Tooltip>
+
+          <Tooltip title='Combine selected cells'>
+            <Button type='dashed' icon={<MergeCellsOutlined />} size='small' />
+          </Tooltip> */}
```

Comment:
next time, don't add this functionality until on the particular ticket itself for it